packages/ui/src/ui/utils/tablet_cell_bundles/bundles/bundle-editor-dialog/index.ts

```diff
@@ -269,6 +270,25 @@ export const simpleBundleValidate = (v?: number | string) => {
     return error;
 };
 
+export const dcBundleValidate = ({
```

Comment:
I think it is better to name it like `instanceCountValidate`/`validateInstanceCount`/...

Reply:
f2cf7f75c35c653d9d499b488f7df424006bff8f